model/modelprocessor/errormessage.go

```diff
@@ -32,7 +32,10 @@ func (s SetErrorMessage) ProcessBatch(ctx context.Context, b *modelpb.Batch) err
 	for i := range *b {
 		event := (*b)[i]
 		if event.Error != nil {
-			event.Message = s.setErrorMessage(event)
+			event.Error.Message = s.setErrorMessage(event)
```

Comment:
> Perhaps the simplest thing to do would be to change SetErrorMessage to only set message if it's empty.

My understanding of @axw 's comment is that we don't touch event.Error.Message, and only set event.Message if it is empty.

Reply:
Okay, then I'm miss understanding the purpose of `event.Error.Message`. Why would we not capture the error body in an error field, but potential capture it in a non-error field?

Reply:
@rubvs the error message is already assigned to `event.Error.Message` in https://github.com/elastic/apm-data/blob/main/input/elasticapm/internal/modeldecoder/v2/decoder.go#L1690